atm/src/atm-states/WithdrawCashAtmState.ts

```
@@ -0,0 +1,37 @@
+
+export class WithdrawCashAtmState extends AtmState {
+    private isValidAmount(amount: number) {
+        if (amount <= 0) return false;
+        if (amount % 100 !== 0) return false;
+        return true;
+    }
+
+    withdrawCash(amount: number): void {
+        if (!this.isValidAmount(amount)) {
+            console.log('Entered amount must be positive and a multiple of 100!');
+            return this.exit();
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)